### PR TITLE
Added Sum function

### PR DESCRIPTION
**Pull request by AbhijithRavindran**
Simple sum function added

### CODE REVIEW

**Comment by AbhijithSpericorn** [comment]
Thank you :+1:  :)

File: helloworld.rb [hunk 1]
```diff
@@ -1 +1,4 @@
-puts "Hello world!"
\ No newline at end of file
+puts "Hello world!"
+a=5
+b=2
+puts  "Sum of a and b is #{a+b}"
```
Comment (AbhijithSpericorn) [inline helloworld.rb]: Very good

Comment (AbhijithRavindran) [reply helloworld.rb]: Thanks

**Comment by AbhijithRavindran** [comment]
Thankyou for approval

**Pull request merged**